Makefile

```diff
@@ -52,6 +52,7 @@ IMG ?= controller:latest
 ENVTEST_K8S_VERSION = 1.23
 
 CERT_MANAGER_VERSION = v1.8.0
+PWD=$(shell pwd)
```

Comment:
Is this a leftover? I don't see where you're using it

Reply:
yes